### PR TITLE
feat: move "next" optimize docs into new optimize instance (subset #1)

### PR DESCRIPTION
**Pull request by pepopowitz**
## What is the purpose of the change

Moves a subset of the "next" optimize docs into the new optimize instance

This subset contains everything outside of a cluster of docs that are very intertwined, which will come in a separate PR. 

Docs included in this PR: 
- `optimize/userguide/*` (except `creating-reports`, which is part of the network of tightly-coupled docs)
- `optimize/userguide/additional-features/*` (except `alerts`, which is part of the network of tightly-coupled docs)
- `optimize/dashboards/*` (which doesn't technically exist as a navigable path for the users, but correlates to a single `Dashboards` section in the left navigation)

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by pepopowitz** [comment]
heads up friends that now that #1166 has merged, this is ready for final review

**Review by christinaausley — APPROVED** [review]
Nice work here. I don't see this impacting any open PRs. Looks good from a technical documentation perspective!

**Review by akeller — APPROVED** [review]

**Comment by akeller** [comment]
@pepopowitz I'll let you merge this when you are ready.

**Pull request merged**
